source/includes/list-table-shell-properties.rst

```diff
@@ -103,12 +103,16 @@
    * - ``redactHistory``
      - string
      - ``remove``
-     - Controls what information is recorded in the shell history.
+     - Controls what information is recorded in the ``mongosh`` command history, 
+       meaning what can be accessed via up and down keys and Ctrl+R while inside the shell. 
```

Comment:
[suggestion] Instead of using a dependent clause here, I think we can just split this up into separate sentences to make it more concise/simple. Maybe something like: 
```
Controls what information is recorded in the `mongosh` command history. You can access the `mongosh` command history via the up and down...
```